start/sw/mobile-app/mobile-installation.md

```diff
@@ -8,20 +8,24 @@ description: Creating mobile applications using Google Android Studio.
 
 # Mobile application
 
-Today wide range of smart mobile devices make it possible to create clever applications. 
+Today wide range of smart mobile devices make it possible to create clever applications.
 
-# Get started
+## Get started
 
-## Install required libraries
+### Install required libraries
 
-`sudo dnf install glibc.i686 glibc-devel.i686 libstdc++.i686 zlib-devel.i686 ncurses-devel.i686`
+```$ sudo dnf install glibc.i686 glibc-devel.i686 libstdc++.i686 zlib-devel.i686 ncurses-devel.i686```
```

Comment:
Does this work fine when you lose the `.i686` suffix? That is (unnecessarily) architecture specific...